### PR TITLE
Add: Requirements for 'make verify' command

### PR DESCRIPTION
**Pull request by andygout**

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint no-console: 0 */
```
Comment (ironsidevsquincy) [inline server/app.js]: you can use n-logger

File: client/main.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 window.onload = function () {
 
+	let d3;
```
Comment (ironsidevsquincy) [inline client/main.js]: what's this for? looks like `d3` is a global?

Comment (andygout) [inline client/main.js]: `error  'd3' is not defined  no-undef`

Comment (ironsidevsquincy) [inline client/main.js]: Right, so tell eslint it's a global, i.e. defined somewhere else

http://eslint.org/docs/user-guide/configuring#specifying-globals

I'm surprised this still works, would have thought this `let` would overwrite the global...

Comment (andygout) [inline client/main.js]: `/* global d3 */` - will add these fixes on separate branch.

**Pull request merged**